server/spec/content_access_spec.rb

```diff
@@ -682,4 +687,95 @@ def read_json_file(filename)
+        @owner['key'], "cname-c2", 'test-content-c2', random_string("clabel"), "ctype", "cvendor",
+        {:content_url=> '/this/is/the/path',  :modified_products => [@modified_product["id"]]}, true)
+    @cp.add_content_to_product(@owner['key'], product_2['id'], content_c2['id'], true)
+
+    create_pool_and_subscription(@owner['key'], product_2['id'], 10)
+
+    @consumer = consumer_client(@user, @consumername, type=:system, username=nil,
+      facts= {'system.certificate_version' => '3.3'})
+    certs = @consumer.list_certificates
+
+    expect(certs.length).to eq(1)
+
+    cert = certs[0]['cert']
+    json_body = extract_payload(cert)
+    expect(json_body['products'][0]['content'].length).to eq(2)
+
+    # Make sure that content c1 is not present in cert,
+    # since product_1 does not have active pool
+    json_body['products'][0]['content'].each do |content|
+      expect(content.id).to_not eq(content_c1.id)
+    end
+  end
+
+  it 'should include content from all products associated with active pool to SCA cert' do
+    mkt_product1 = create_product(random_string('productp1'), random_string('product'),
+      {:owner => @owner['key']})
+    eng_product = create_product(random_string('productp2'), random_string('product'),
+      {:owner => @owner['key']})
+    derived_product =create_product(random_string('productp4'), random_string('product'),
+      {:owner => @owner['key']})
+    dev_eng_product = create_product(random_string('productp3'), random_string('product'),
+      {:owner => @owner['key']})
+
+    # Content enabled = true
+    content_c1 = @cp.create_content(
+        @owner['key'], "content_c1", 'test-content-c1', random_string("clabel"), "ctype", "cvendor",
+        {:content_url=> '/this/is/the/path',  :modified_products => [@modified_product["id"]]}, true)
+    @cp.add_content_to_product(@owner['key'], eng_product['id'], content_c1['id'], true)
+
+    content_c2 = @cp.create_content(
+        @owner['key'], "content_c2", 'test-content-c2', random_string("clabel"), "ctype", "cvendor",
+        {:content_url=> '/this/is/the/path',  :modified_products => [@modified_product["id"]]}, true)
+    @cp.add_content_to_product(@owner['key'], mkt_product1['id'], content_c2['id'], true)
+
+    content_c3 = @cp.create_content(
+        @owner['key'], "content_c3", 'test-content-c3', random_string("clabel"), "ctype", "cvendor",
+        {:content_url=> '/this/is/the/path',  :modified_products => [@modified_product["id"]]}, true)
+    @cp.add_content_to_product(@owner['key'], derived_product['id'], content_c3['id'], true)
+
+    content_c4 = @cp.create_content(
+        @owner['key'], "content_c4", 'test-content-c4', random_string("clabel"), "ctype", "cvendor",
+        {:content_url=> '/this/is/the/path',  :modified_products => [@modified_product["id"]]}, true)
+    @cp.add_content_to_product(@owner['key'], dev_eng_product['id'], content_c4['id'], true)
+
+
+    create_pool_and_subscription(@owner['key'], mkt_product1.id,
```

Comment:
We should try to be consistent with the way we do line continuation and indentation. This statement here trails off early and is partially indented on the third line of the statement, whereas the statement on line 759 below is heavily indented immediately, and the statements above on line 739, 744, and 749 are all indented with 4 spaces on each line.

This kind of inconsistency destroys auto-alignment detection in IDEs, and makes it harder to read at a glance. Let's pick one level of indentation (either 4 or 2 spaces), and then follow a single pattern for indentation and line continuation.

Reply:
Noted. Corrected the indentation.